ajax/libs/material-components-web/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "material-components-web",
+  "version": "0.11.1",
+  "filename": "material-components-web.min.js",
+  "description": "Modular and customizable Material Design UI components for the web",
+  "homepage": "https://material.io/components/",
+  "keywords": [
+    "material components",
+    "material design"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Travis Kaufman <@traviskaufman>",
```

Comment:
I think the author is addyosmani, addyosmani@gmail.com, https://www.addyosmani.com
He is the first one who created this repo.
ref: https://github.com/material-components/material-components-web/graphs/contributors?from=2014-04-06&to=2014-04-18&type=c

Reply:
To allow this PR to progress, I believe it would be best to just use `"The Material Design Team"` to keep it open and less specific to a single author?